core/core_bind.cpp

```diff
@@ -257,6 +257,15 @@ Error OS::shell_open(String p_uri) {
 	return ::OS::get_singleton()->shell_open(p_uri);
 }
 
+Error OS::shell_show_in_explorer(String p_path, bool p_open_folder) {
+	if (p_path.begins_with("res://")) {
+		WARN_PRINT("Attempting to explore file path with the \"res://\" protocol. Use `ProjectSettings.globalize_path()` to convert a Godot-specific path to a system path before opening it with `OS.shell_show_in_explorer()`.");
+	} else if (p_path.begins_with("user://")) {
+		WARN_PRINT("Attempting to explore file path with the \"user://\" protocol. Use `ProjectSettings.globalize_path()` to convert a Godot-specific path to a system path before opening it with `OS.shell_show_in_explorer()`.");
+	}
```

Comment:
Shouldn't this be wrapped in `#ifdef DEBUG_ENABLED`?

Reply:
I'm not sure about this, but `shell_open()` is not wrapped in this macros, too.